data/collections/RS500.json

```diff
@@ -486,7 +486,7 @@
             "year": 1991,
             "appleMusicURL": null,
             "appleMusicID": null,
-            "imageURL": "https:\/\/www.rollingstone.com\/wp-content\/uploads\/2020\/09\/R1344-054-James-Brown-Star-Time.jpg"
+            "imageURL": null
```

Comment:
Might be worth keeping the URLs if they don't exist in Apple Music? Or could even host the somewhere?

Reply:
What about copyright?

Reply:
Fair call. That's always a risk with album artwork I suppose. Safer to not host it.

(Though I have some in the screenshots anyway already.)